GTG/gtk/editor/taskeditor.ui

```diff
@@ -4,403 +4,461 @@
   <object class="GtkWindow" id="TaskEditor">
     <property name="can_focus">False</property>
     <property name="title" translatable="yes">Task</property>
-    <property name="default_width">500</property>
-    <property name="default_height">400</property>
+    <property name="default_width">800</property>
+    <property name="default_height">500</property>
```

Comment:
Sure thing, the reason for bigger size was that I considered the view less cluttered that way, however 500x400 does not affect functionality or visibility of widgets, so we can live with that :)